### PR TITLE
Update documentation to match changes of IGN

### PR DESCRIPTION
**Pull request by babastienne**
## Checklist

- [x] I have followed the guidelines in our [Contributing document](https://geotrek.readthedocs.io/en/latest/CONTRIBUTING.html)
- [x] My code respects the Definition of done available in the [Development section of the documentation](https://geotrek.readthedocs.io/en/latest/contribute/development.html#definition-of-done-for-new-model-fields)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] ~~I have added tests that prove my fix is effective or that my feature works.~~
- [ ] ~~New and existing unit tests pass locally with my changes~~
- [x] I added an entry in the changelog file
- [ ] ~~My commits are all using prefix convention (emoji + tag name) and references associated issues~~
- [x] I added a label to the PR corresponding to the perimeter of my contribution
- [ ] ~~The title of my PR mentionned the issue associated~~

### CODE REVIEW

File: docs/install/advanced-configuration.rst [hunk 1]
```diff
@@ -172,25 +172,7 @@ Map settings
...
-         '&copy; IGN - GeoPortail'),
-        ('OSM', 'http://{s}.tile.osm.org/{z}/{x}/{y}.png', '&copy; OSM contributors'),
-    ]
-
-    LEAFLET_CONFIG['OVERLAYS'] = [
-        ('Cadastre',
-         '//wxs.ign.fr/<key>/wmts?LAYER=CADASTRALPARCELS.PARCELS&EXCEPTIONS=text/xml&FORMAT=image/png&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=bdparcellaire_o&TILEMATRIXSET=PM&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}',
-         '&copy; IGN - GeoPortail'),
-    ]
+By default, you have 2 basemaps layers in your Geotrek-admin (OSM and OpenTopoMap):
```
Comment (babastienne) [inline docs/install/advanced-configuration.rst:175]:
```suggestion
By default, you have 2 basemaps layers in your Geotrek-admin (OSM and OpenTopoMap)
```

File: docs/install/advanced-configuration.rst [hunk 2]
```diff
@@ -211,18 +193,82 @@ Example with IGN and OSM basemaps:
...
+            '//{s}.tile.opentopomap.org/{z}/{x}/{y}.png',
+            {
+                'attribution': 'map data: © <a href="https://openstreetmap.org/copyright">OpenStreetMap</a> contributors, <a href="http://viewfinderpanoramas.org">SRTM</a> | map style: © <a href="https://opentopomap.org">OpenTopoMap</a> (<a href="https://creativecommons.org/licenses/by-sa/3.0/">CC-BY-SA</a>)',
+                'maxNativeZoom': 17,
+                'maxZoom': 22
+            }
+        ),
+        (
+            'IGN Scan 25',
+            '//data.geopf.fr/private/wmts?SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetCapabilities&apikey=ign_scan_ws&LAYER=GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN25TOUR&STYLE=normal&FORMAT=image/jpeg&TILEMATRIXSET=PM&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}',
```
Comment (babastienne) [inline docs/install/advanced-configuration.rst:244]:
```suggestion
            '//data.geopf.fr/private/wmts?apikey=ign_scan_ws&LAYER=GEOGRAPHICALGRIDSYSTEMS.MAPS&EXCEPTIONS=text/xml&FORMAT=image/jpeg&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=normal&TILEMATRIXSET=PM&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}',
```

File: docs/install/advanced-configuration.rst [hunk 3]
```diff
@@ -211,18 +193,82 @@ Example with IGN and OSM basemaps:
...
+                'attribution': 'Plan Scan 25 Touristique - Carte © IGN/Geoportail',
+                'maxNativeZoom': 19,
+                'maxZoom': 22
+            }
+        ),
     ]
 
 To use some IGN Geoportail WMTS tiles (Scan25, Scan100, etc.), you may need an API key. You can find more information about this on https://geoservices.ign.fr/services-web-issus-des-scans-ign.
 
+You can alos configure overlays layers like this :
```
Comment (babastienne) [inline docs/install/advanced-configuration.rst:255]:
```suggestion
You can also configure overlays layers like this :
```

File: tools/check_ign_key.sh [hunk 4]
```diff
@@ -8,21 +8,14 @@ if test -z "$referer" -o -z "$key"; then
     exit 1
 fi
 
-layers_jpeg="GEOGRAPHICALGRIDSYSTEMS.MAPS GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN25TOUR ORTHOIMAGERY.ORTHOPHOTOS"
-layers_png="CADASTRALPARCELS.PARCELS"
+layers_jpeg="GEOGRAPHICALGRIDSYSTEMS.MAPS GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN25TOUR"
```
Comment (babastienne) [inline tools/check_ign_key.sh:11]:
```suggestion
layers_jpeg="GEOGRAPHICALGRIDSYSTEMS.MAPS GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN25TOUR SCAN100_PYR-JPEG_WLD_WM_WMTS_3D"
```

File: tools/check_ign_key.sh [hunk 5]
```diff
@@ -8,21 +8,14 @@ if test -z "$referer" -o -z "$key"; then
...
 
 echo '------------------------------------------------------------'
 echo GetCapabilities
-curl -s -I --header "Referer: $referer" "http://wxs.ign.fr/$key/wmts?SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetCapabilities" | head -n 1
+curl -s -I --header "Referer: $referer" "https://data.geopf.fr/private/wmts?SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetCapabilities&apikey=$key" | head -n 1
 
 for layer in $layers_jpeg; do
     echo '------------------------------------------------------------'
     echo $layer
     curl -s -I --header "Referer: $referer" "https://wxs.ign.fr/$key/wmts?LAYER=$layer&EXCEPTIONS=text/xml&FORMAT=image/jpeg&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=normal&TILEMATRIXSET=PM&TILEMATRIX=12&TILEROW=1478&TILECOL=2112" | head -n 1
```
Comment (babastienne) [inline tools/check_ign_key.sh:20]:
```suggestion
    curl -s -I --header "Referer: $referer" "https://data.geopf.fr/private/wmts?apikey=$key&LAYER=$layer&EXCEPTIONS=text/xml&FORMAT=image/jpeg&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=normal&TILEMATRIXSET=PM&TILEMATRIX=12&TILEROW=1478&TILECOL=2112" | head -n 1
```

**Comment by babastienne** [comment]
@submarcos je veux bien un coup de main sur cette PR ? Pas certain de bien voir pourquoi il y a des tests qui sont KO. Si jamais tu as quelques instants (toi ou quelqu'un d'autre) je veux bien que vous repreniez ce travail pendant mon absence.

Merci :pray:

**Comment by submarcos** [comment]
@babastienne c'est surement le changement d'ordre des fonds de plan le soucis.

Il vaut mieux separer les modifs, une PR pour la doc, une pour les trads, une pour l'ordre par defaut des fonds de plan

File: geotrek/settings/base.py [hunk 6]
```diff
@@ -447,10 +447,24 @@ def api_bbox(bbox, buffer):
 LEAFLET_CONFIG = {
     'SRID': 3857,
     'TILES': [
-        ('OpenTopoMap', 'https://{s}.tile.opentopomap.org/{z}/{x}/{y}.png',
-         {'attribution': 'Données: © Contributeurs OpenStreetMap, SRTM | Affichage: © OpenTopoMap (CC-BY-SA)',
-          'maxZoom': 17}),
-        ('OSM', 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', '© Contributeurs OpenStreetMap'),
+        (
```
Comment (marcantoinedupre) [inline geotrek/settings/base.py:450]: Is there a reason for changing the order? IMO OpenTopoMap is a relevant default tile layer.

**Pull request merged**